src/Core/Resolvers/CosmosMutationEngine.cs

```diff
@@ -234,6 +255,118 @@ private static async Task<ItemResponse<JObject>> HandleUpdateAsync(IDictionary<s
             }
         }
 
+        /// <summary>
+        /// Refer https://learn.microsoft.com/en-us/azure/cosmos-db/partial-document-update for more details on patch operations
+        /// </summary>
+        /// <param name="queryArgs"></param>
+        /// <param name="container"></param>
+        /// <returns></returns>
+        /// <exception cref="InvalidDataException"></exception>
+        /// <exception cref="DataApiBuilderException"></exception>
+        private static async Task<JObject> HandlePatchAsync(IDictionary<string, object?> queryArgs, Container container)
```

Comment:
```suggestion
        /// <summary>
        /// Refer to https://learn.microsoft.com/azure/cosmos-db/partial-document-update for more details on patch operations
        /// </summary>
        /// <param name="queryArgs"></param>
        /// <param name="container"></param>
        /// <returns></returns>
        /// <exception cref="InvalidDataException"></exception>
        /// <exception cref="DataApiBuilderException"></exception>
        private static async Task<JObject> HandlePatchAsync(IDictionary<string, object?> queryArgs, Container container)
```
incomplete param/exception comments